package.json

```diff
@@ -22,7 +22,7 @@
     "eslint-config-airbnb": "^18.2.1",
     "eslint-plugin-import": "^2.22.1",
     "import-webflow-export": "^0.0.4",
-    "municipal-money-charts": "https://github.com/OpenUpSA/municipal-money-charts.git#overlay-chart-missing-bar",
+    "municipal-money-charts": "https://github.com/OpenUpSA/municipal-money-charts.git#comparison-click",
```

Comment:
remember to update this to the hash in master once we merge the change, because we delete branches after merging them normally